server/src/DbtProfileCreator.ts

```diff
@@ -101,17 +98,20 @@ export class DbtProfileCreator {
     const targetConfig = profile.outputs[target];
     const { type, method } = targetConfig;
 
-    const profileBuilder = BIG_QUERY_PROFILES.get(method);
-    if (!profileBuilder) {
-      throw new Error(`Unknown authentication method of '${type}' profile`);
-    }
+    let dbtProfile: DbtProfile | undefined = undefined;
 
-    const dbtProfile = profileBuilder();
+    if (type.valueOf() === DbtProfileType.BigQuery) {
```

Comment:
Shouldn't we do the same with right side `DbtProfileType.BigQuery.valueOf()`?

Reply:
Seems not